Class-10-(Scope-Exceptions-Types).md

```diff
@@ -0,0 +1,23 @@
+## In the prep work this week, the students learned:
+1. More about scope. Scope was covered briefly in the Functions chapter, however, now students have covered it more in-depth in terms of the order code executes in.
```

Comment:
Rephrasing suggestion: "Scope was covered briefly in the Functions chapter, but the students have now covered it in more depth."

Reply:
Done! Thanks!